datastore/google/cloud/datastore/transaction.py

```diff
@@ -234,3 +282,36 @@ def commit(self):
         finally:
             # Clear our own ID in case this gets accidentally reused.
             self._id = None
+
+    def put(self, entity):
+        """Remember an entity's state to be saved during :meth:`commit`.
+
+        .. note::
+           Any existing properties for the entity will be replaced by those
```

Comment:
It's typically "good form" with Sphinx to have an empty line between the `.. note::` directive and the actual note content. (I realize you copy-pasted from somewhere else, just worth pointing out for your future usage. I also realize other contributors don't follow this / aren't aware of this, but still worth sharing.)

Reply:
ok.